CHANGELOG.md

```diff
@@ -3,6 +3,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [2.5.7] - 2025-06-10
+### :bug: Fixed
+- Fix connection leak issue due to Limitless Plugin not properly closing off monitoring connections ([PR #1369](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1369)).
+- Fix leaked connection in Limitless plugin ([PR #1388](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1388)).
+- Fix wrong calls in wrapper class ([PR #1413](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1413)).
+- Fix host info type ([PR #1423](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1423)).
+
+### :crab: Changed
+- Javadoc improvement ([PR #1368](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1368)).
+- Change the storage type for multi-az clusters ([PR #1392](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1392)).
+- Set `failover2` as default plugin ([PR #1414](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1414)).
+- Improve logging for Limitless Plugin ([PR #1431](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1431)).
+- Clarifies support for blue/green deployments by specifying minimum versions ([PR #1434](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1434)).
+
+### :magic_wand: Added
+- Add support of Blue/Green Deployment for Aurora Mysql cluster, Aurora Postgres cluster, RDS Mysql Instance, RDS Postgres Instance ([PR #1355](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1355)).
```

Comment:
Lets link the docs instead of the pr: https://github.com/aws/aws-advanced-jdbc-wrapper/blob/main/docs/using-the-jdbc-driver/using-plugins/UsingTheBlueGreenPlugin.md